### PR TITLE
Update_4G-router_guidance_reset_info

### PR DESCRIPTION
**Pull request by amylaurence1**
### Context
Updated information on resetting routers after 31 July 2021.

### CODE REVIEW

File: app/views/devices_guidance/4g_user_guidance.md [hunk 1]
```diff
@@ -1,53 +1,59 @@
-For all router types, users will find the following in the box:
+<div class="govuk-inset-text">
+  <p>
+    All 4G wireless routers come with SIM cards that provide data until 31 July 2021. To continue using routers after this date, you must <a href="/devices/reset-your-wireless-router" class="govuk-link">reset them and replace their SIM cards</a>.
+  </p>
+</div>
+
+You should make young people and families aware that:
+
+- SIM cards will work until the 31 July 2021
```
Comment (Lloyd-K) [inline app/views/devices_guidance/4g_user_guidance.md:9]: Suggest removing 'the'

File: app/views/devices_guidance/4g_user_guidance.md [hunk 2]
```diff
@@ -1,53 +1,59 @@
...
 
-- WiFi network name (SSID)
-- Password (this is case sensitive)
+- wifi network name (SSID)
+- password (this is case sensitive)
 
 ## Technical support
 
-Children and families should raise any queries regarding 4G wireless routers with the trust, local authority or school which sent the device to them. They will be able to support with setting up routers or raising an issue if there is a problem with the router or SIM card.  They should not press the reset button.
+For technical support, children and families should contact the trust, local authority or school which sent them the router. They can help with setting up routers or raising an issue if there is a problem with the router or SIM card.
```
Comment (Lloyd-K) [inline app/views/devices_guidance/4g_user_guidance.md:56]: there's

File: app/views/devices_guidance/4g_user_guidance.md [hunk 3]
```diff
@@ -63,6 +69,15 @@ You can check the status of each router in the [Support Portal](https://computac
...
+- [Safeguarding and remote education during coronavirus](https://www.gov.uk/guidance/safeguarding-and-remote-education-during-coronavirus-covid-19) - guidance to help you understand safeguarding procedures when planning remote education strategies and teaching remotely.
+- [Support for parents and carers to keep children safe from online harm](https://www.gov.uk/government/publications/coronavirus-covid-19-keeping-children-safe-online/coronavirus-covid-19-support-for-parents-and-carers-to-keep-children-safe-online) - outlines resources to help keep children safe from different risks online, and where to go to receive support and advice.
+- [Support to stay safe online](https://www.gov.uk/guidance/covid-19-staying-safe-online) - includes information on security and privacy settings.
+
+In conversations with parents, carers, and children you should emphasise the importance of a safe online environment and offer support and advice on how to do this. It is also important that children, parents, carers, and school staff are clear on what the reporting mechanisms are if they have any concerns regarding online safety.
```
Comment (Lloyd-K) [inline app/views/devices_guidance/4g_user_guidance.md:81]: how to create this.

It's 

... clear on how to report any concerns they may have about online safety.

**Review by Lloyd-K — CHANGES_REQUESTED** [review]
I've suggested a couple of small changes.

**Review by markhallen — APPROVED** [review]

**Review by markhallen — APPROVED** [review]

**Pull request merged**